### PR TITLE
Change doc title in dev process

### PR DESCRIPTION
**Pull request by cristianemoyano**

### CODE REVIEW

**Review by Franavarro — APPROVED** [review]
Look good to me

**Review by jeremias-adm — COMMENTED** [review]
LGMT

**Review by jeremias-adm — APPROVED** [review]
LGMT

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ git clone https://github.com/cristianemoyano/math_utils.git
 cd math_utils
 ```
 
-### New change
+### How to add a new change
```
Comment (Franavarro) [inline README.md:28]: This is a test

**Pull request merged**